utils/experiment/experiment.go

```diff
@@ -18,6 +18,14 @@ import (
 
 var terminateExperimentPatch = []byte(`{"spec":{"terminate":true}}`)
 
+func BelongsToRollout(experiment *v1alpha1.Experiment) bool {
+	if len(experiment.OwnerReferences) == 0 {
+		return false
+	}
+
+	return experiment.OwnerReferences[0].Kind == "Rollout"
```

Comment:
Even though I do not know of any use cases where an Experiment has multiple parents the k8s spec does allow it, can we loop through them here instead of using '[0]'

Reply:
updated. lmk if this looks good to you